S/SIMDe/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message
+using BinaryBuilder, Pkg
+
+name = "SIMDe"
+version = v"0.7.6"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource(
+        "https://github.com/simd-everywhere/simde.git",
+        "fefc7857ff3e785b988a61a8f5f3c5bd5eb24342",
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/simde
+install_license COPYING
+mkdir build
+cd build
+meson .. --prefix=${prefix} --libdir=${libdir} --buildtype=release --cross-file=${MESON_TARGET_TOOLCHAIN} -Dtests=false
+ninja install
```

Comment:
Please always use `-j${nproc}` to avoid oversubscribing the machine
```suggestion
ninja -j${nproc} install
```